src/resources/index.ts

```diff
@@ -11,6 +11,11 @@ const Relation = mongoose.model('relations', relationSchema);
 
 export default async function(app: Express) {
 
+  function fixId(doc: any) {
```

Comment:
I'm fairly new to express, is there a better place to store utility functions?

Reply:
Not really, this is fine.

Reply:
Sometimes I aggregate them elsewhere in utils file or directory and import them, but this is fine.